upup/pkg/fi/cloudup/awstasks/network_load_balancer.go

```diff
@@ -680,15 +706,22 @@ func (_ *NetworkLoadBalancer) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *Ne
 }
 
 type terraformNetworkLoadBalancer struct {
-	Name                   string               `json:"name" cty:"name"`
-	Internal               bool                 `json:"internal" cty:"internal"`
-	Type                   string               `json:"load_balancer_type" cty:"load_balancer_type"`
-	Subnets                []*terraform.Literal `json:"subnets" cty:"subnets"`
-	CrossZoneLoadBalancing bool                 `json:"enable_cross_zone_load_balancing" cty:"enable_cross_zone_load_balancing"`
+	Name                   string                                      `json:"name" cty:"name"`
+	Internal               bool                                        `json:"internal" cty:"internal"`
+	Type                   string                                      `json:"load_balancer_type" cty:"load_balancer_type"`
+	Subnets                []*terraform.Literal                        `json:"subnets,omitempty" cty:"subnets"`
```

Comment:
Ah I just realized we're removing the Subnets field from the `NetworkLoadBalancer` struct above. In that case we can just remove `Subnets` from here and Cloudformation, and remove `omitempty` from the SubnetMappings json field tags since we should always be setting it.

Reply:
Ah true. I've removed the Subnets field now.